docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -283,13 +285,13 @@ For example, to filter the results of the `SelectMedia()` method of the [Tizen.C
     ```
 
 
-## Finding Folders
+## Find folders
 
-To find media folders:
+To find media folders, follow the steps below:
 
 1.  To find media folders and filter the results, use the `Select()` method of the [Tizen.Content.MediaContent.FolderCommand](/application/dotnet/api/TizenFX/latest/api/Tizen.Content.MediaContent.FolderCommand.html) class.
 
-    The following example filters media folders so that only folders named "Downloads" found in the internal storage are included in the result. For more information on the filter properties, see [Setting up a Filter](#setting-up-a-filter).
+    The following example filters media folders so that only folders named "Downloads" found in the internal storage are included in the result. For more information on the filter properties, see [Set up a Filter](#set-up-a-filter).
```

Comment:
```suggestion
    The following example filters media folders so that only folders named "Downloads" found in the internal storage are included in the result. For more information on the filter properties, see [set up a filter](#set-up-a-filter).
```